package.json

```diff
@@ -14,7 +14,7 @@
     "hubot"
   ],
   "dependencies": {
-    "hubot": ">= 2.6.0 < 3"
+    "hubot": ">=2.6.0 <10 || 0.0.0-development"
```

Comment:
I’m not sure ranges like that are supported in `dependencies`, I think they are rather meant for `peerDependencies`? I would update it to `^3.0.0` /cc @technicalpickles